projects/tsq-gallery/src/lib/components/tsq-gallery-viewer/tsq-gallery-viewer.component.html

```diff
@@ -1,50 +1,50 @@
 <ng-container *ngIf="isViewerOpen">
   <div id="tsq-gallery-viewer"
-      tabindex="0"
-      [ngStyle]="{'display': isViewerOpen ? 'flex' : 'none', 'cursor': isMoving? 'grabbing' : 'default'}"
-      (keydown)="onKeyDown($event)"
-      (mouseup)="dragUp()"
-      (mouseleave)="dragUp()"
-      (touchstart)="touchStart($event)"
-      (touchmove)="touchMove($event)"
-      (touchend)="touchEnd()"
-      (click)="backdropClickClose && !isMoving && !showLoading && close()"
-      #backdrop
-      [@fadeInOut]>
+       tabindex="0"
+       [ngStyle]="{'display': isViewerOpen ? 'flex' : 'none', 'cursor': isMoving? 'grabbing' : 'default'}"
+       (keydown)="onKeyDown($event)"
+       (mouseup)="dragUp()"
+       (mouseleave)="dragUp()"
+       (touchstart)="touchStart($event)"
+       (touchmove)="touchMove($event)"
+       (touchend)="touchEnd()"
+       (click)="$event.stopPropagation(); backdropClickClose && !isMoving && !showLoading && close()"
```

Comment:
This is really complex to a (click) inliner, should be extracted to it's own function